test/new-e2e/tests/containers/ecs_test.go

```diff
@@ -257,15 +258,13 @@ func (suite *ecsSuite) TestRedisECS() {
 				`^image_id:sha256:`,
 				`^image_name:public.ecr.aws/docker/library/redis$`,
 				`^image_tag:latest$`,
-				`^redis_host:`,
-				`^redis_port:6379$`,
-				`^redis_role:master$`,
 				`^short_image:redis$`,
 				`^task_arn:`,
 				`^task_family:.*-redis-ec2$`,
 				`^task_name:.*-redis-ec2$`,
 				`^task_version:[[:digit:]]+$`,
 			},
+			AcceptUnexpectedTags: true,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Wonder if you should allow for a certain pattern of tags, here for instance keys with `redis*`. We should catch reserved tags such as `host` https://docs.datadoghq.com/getting_started/tagging/

Reply:
If we had a clear way to name tags to know what is their source, I would have agreed with you.
The problem is that there isn’t.
For ex., in the context of **incident-25306**, DataDog/integrations-core#16797 added a `port` tag to `nginx` metrics. This tag doesn’t follow any pattern and isn’t prefixed by `nginx_`.
So, in this case, I can’t see how I could have restricted what are “acceptable unexpected tags” to make `port` accepted but not something else.

Reply:
Indeed, only metrics have namespaces. For customers this narrows down the tags enough that I guess nobody felt the need to ask for namespaces on tags too.